docs/versioned_docs/version-0.2.x/how_to/llm_token_usage_tracking.ipynb

```diff
@@ -2,169 +2,228 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "id": "e5715368",
+   "id": "90dff237-bc28-4185-a2c0-d5203bbdeacd",
    "metadata": {},
    "source": [
     "# How to track token usage for LLMs\n",
     "\n",
-    "This notebook goes over how to track your token usage for specific calls. It is currently only implemented for the OpenAI API.\n",
+    "Tracking token usage to calculate cost is an important part of putting your app in production. This guide goes over how to obtain this information from your LangChain model calls.\n",
     "\n",
-    "Let's first look at an extremely simple example of tracking token usage for a single LLM call."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 1,
-   "id": "9455db35",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "from langchain_community.callbacks import get_openai_callback\n",
-    "from langchain_openai import OpenAI"
+    "```{=mdx}\n",
+    "import PrerequisiteLinks from \"@theme/PrerequisiteLinks\";\n",
+    "\n",
+    "<PrerequisiteLinks content={`\n",
+    "- [LLMs](/docs/concepts/#llms)\n",
+    "`} />\n",
+    "```\n",
+    "\n",
+    "## Using LangSmith\n",
+    "\n",
+    "You can use [LangSmith](https://www.langchain.com/langsmith) to help track token usage in your LLM application. See the [LangSmith quick start guide](https://docs.smith.langchain.com/).\n",
+    "\n",
+    "## Using callbacks\n",
+    "\n",
+    "There are some API-specific callback context managers that allow you to track token usage across multiple calls. You'll need to check whether such an integration is available for your particular model.\n",
+    "\n",
+    "If such an integration is not available for your model, you can create a custom callback manager by adapting the implementation of the OpenAI callback manager (find it in the code base and adapt for your own LLMs.)\n",
```

Comment:
```suggestion
    "If such an integration is not available for your model, you can create a custom callback manager by adapting the implementation of the [OpenAI callback manager](https://api.python.langchain.com/en/latest/_modules/langchain_community/callbacks/openai_info.html#OpenAICallbackHandler).\n",
```

Reply:
@ccurme Did you mean to link to the the OpenAI callback handler `OpenAiCallbackHandler` or the OpenAI specific context manager `get_openai_callback()`?

Reply:
Yeah good call that fits the context better: https://api.python.langchain.com/en/latest/_modules/langchain_community/callbacks/manager.html#get_openai_callback